### ISSUE TITLE
Incorrect text colour for attachment tooltips

### ISSUE THREAD

**Issue by weakit** [vednoc/dark-whatsapp#188]
- **Browser**: Firefox 88.0.1
- **WhatsApp Web version**:  2.2119.6
- **Dark-WhatsApp version**: v3.6.1
- **Additional context**: Light text would be more appropriate on a dark background.
- **Screenshot**: 
![image](https://user-images.githubusercontent.com/34541656/120481997-4f892780-c3ce-11eb-9427-f28c0e5a30b7.png)

**Comment by kyle-ds** [comment]
The selector is ```_3u6fL```.

I'm not entirely sure how to modify the userstyle to add this, or I would.

**Comment by vednoc** [comment]
@weakit I apologize for replying so late, my notifications have been a complete mess over the last few months: they were being automatically set to read for some reason.

Hi @kyle-ds, thank you for pinging me otherwise I wouldn't have seen this issue.

It seems like they've switched to CSS variables, so there's no need for class selectors. You can fix it by adding the following snippet after [line 862](https://github.com/vednoc/dark-whatsapp/blob/3.6.1/wa.user.styl#L862). It can be placed anywhere in the `:root` selector, but it's better if it's exactly there in case we ever have a need to diff the code.

```styl
--tooltip-text: var(--fg-1) //hsl(0, 0%, 0%);
--tooltip-text-rgb: var(--fg-1-rgb) //0,0,0;
```

**Comment by weakit** [comment]
@vednoc no probs, happy to help.

**Comment by kyle-ds** [comment]
@vednoc  should I also make the mod in the .css file?

**Comment by vednoc** [comment]
> should I also make the mod in the .css file?

No, just in the `wa.user.styl`. The CSS file is generated manually every release.

> how did you link the selector to the css variable?

You can see it in rules, check the screenshot. I noticed that our selectors are different: probably because WA was updated, so this shows how much better variables are in comparison to class selectors.

Also, WA makes it hard to inspect that element because it's removed when the cursor is moved away from the button, so you have to pause the debugger while hovering over the button and inspect it that way. The thing is, you can't pause the debugger easily (at least in Firefox), so you have to enable some kind of breakpoint: I use _keyboard_ under _Event Listener Breakpoints_ because it's the best one for majority of use-cases. Once it's enabled, and your mouse is over the button, you can press any key on your keyboard (I use ctrl/alt) and it will pause the debugger.

![image](https://user-images.githubusercontent.com/18245694/125124750-e0070600-e0f8-11eb-9782-0b966385330d.png)

**Comment by vednoc** [comment]
Fixed in #190.
